test/docker-compose.yml

```diff
@@ -12,4 +12,10 @@ services:
       - DEBUG=1
```

Comment:
Can we also please add an explicit version to the image above? At least the major version, otherwise tests will sooner or later start failing.